auth_test.go

```diff
@@ -554,7 +554,8 @@ func TestAuthFastSHA256PasswordEmpty(t *testing.T) {
 	authRespEnd := authRespStart + 1 + len(authResp)
 	writtenAuthRespLen := conn.written[authRespStart]
 	writtenAuthResp := conn.written[authRespStart+1 : authRespEnd]
-	if writtenAuthRespLen != 0 {
+	expectedAuthResp := []byte{0}
+	if writtenAuthRespLen != 1 || !bytes.Equal(writtenAuthResp, expectedAuthResp) {
```

Comment:
no `writtenAuthRespLen != 1 || ` here

Reply:
I removed it because `bytes.Equal` compares length and content, and this if-statement body prints only writtenAuthResp, but not writtenAuthRespLen.
I'll add another if statement for writtenAuthRespLen if you want.

Reply:
Ah, I missed writtenAuthRespLen is displayed in the error message.